### ISSUE TITLE
Country Codes Issue

### ISSUE THREAD

**Issue by telecomsxchange** [telecomsxchange/RatesAlertClient#1]
As described by the documentation:

- User Selects/Chooses Country Code e.g +91
- Choose Network (Area Code) e.g 89

There is no way to enter country code, area code freely, it forces to use one pre-defined country code.

It also showing seller name in area code instead which makes the app not function properly.

**Comment by sanjeev-sharma** [comment]
From the below API response screenshot, can you please tell us which field is for country code (+91, +1 etc) and which field is for Area Code (89 etc) ? 

![screen shot 2018-08-16 at 3 08 48 pm](https://user-images.githubusercontent.com/2708212/44201563-2d638380-a167-11e8-90d7-e21b7be2a913.png)

**Comment by telecomsxchange** [comment]
You will need to allow the user to enter the Prefix field in the app form, it will be passed to the API call as entered by the user. the cron job on the backend should notify the user via push notification whenever the price is available.

**Comment by telecomsxchange** [comment]
For example the user sets an alarm for prefix 9198 and price 0.005 . Once its found to be available it should send a push notification that this prefix 9198 is now available via Seller: $vendor_name on the exchange.

**Comment by sanjeev-sharma** [comment]
Right, then what is the use of having the country code and Area code with-in the app? I mean we can simply ask the user to put a prefix (or country code) and threshold amount and cron job will trigger a push notification based on that. Does that make sense?

**Comment by sanjeev-sharma** [comment]
I got you. I misunderstood I think. So prefix is Country Code + Area Code. Right?

**Comment by telecomsxchange** [comment]
Prefix (Country Code) should be just a field that the user will input in it. you can use auto complete js file to automatically show suggested prefixes and countries as the user is inputting the prefix.

**Comment by telecomsxchange** [comment]
Correct, Prefix = Country Code + Area code. 

No need to worry about creating a pre-defined list, just let the user enter their prefix there. to show complete codes as the user is typing the prefix you can use this API example:

https://github.com/telecomsxchange/get_country_codes

**Comment by sanjeev-sharma** [comment]
Thanks, Got it. Will do it.

**Comment by sanjeev-sharma** [comment]
One last question: Suppose I put 919808 prefix and threshold time 0.0400 

When we run the cron job and fetch the rates for prefix 919808, we get multiple rate results from different vendors. So the question is when will we send the alert? When the price returned for any of the vendors is less than the 0.0400 or when it is more than the 0.0400?
